api/client/attach.go

```diff
@@ -20,6 +20,7 @@ func (cli *DockerCli) CmdAttach(args ...string) error {
 	cmd := Cli.Subcmd("attach", []string{"CONTAINER"}, Cli.DockerCommands["attach"].Description, true)
 	noStdin := cmd.Bool([]string{"#nostdin", "-no-stdin"}, false, "Do not attach STDIN")
 	proxy := cmd.Bool([]string{"#sig-proxy", "-sig-proxy"}, true, "Proxy all received signals to the process")
+	escapeKeys := cmd.String([]string{"-escapeKeys"}, "", "Set up escape key sequence")
```

Comment:
why escape keys? why not detach keys? Btw, for flags we don't do camelcase, there should be a hyphen